std/experimental/ndslice/internal.d

```diff
@@ -1,3 +1,4 @@
+deprecated("Please use appropriate Mir project at http://github.com/libmir")
```

Comment:
`mir-std`. BTW, `ndslice` will be completely reworked. Instead of existing numpy-like `ndslice` a new `ndslice` will be provided. 90% of code will be rewritten. New `ndslice`:

1. Will be faster
2. Will be integrated with ranges
3. Will be integrated with algorithms

In addition,
1. new Range API architecture
2. new algorithm implementation and API
3. 3 types of ndslices:
  a. numpy-like: number of strides equals to number of dimensions
  b. raw-contnious (CBLAS-like): number of strides equals to number of dimensions minus one
  c. ideally continuous: number of strides equals to 0.